vars/cloud.groovy

```diff
@@ -211,3 +248,16 @@ void cleanSkopeo() {
 void skopeoCopyRegistryImages(String oldImageName, String newImageName, int retries = 3) {
     sh "skopeo copy --retry-times ${retries} --tls-verify=false --all docker://${oldImageName} docker://${newImageName}"
 }
+
+/*
+* Get reduced tag, aka X.Y, from the given tag
+*/
+String getReducedTag(String originalTag) {
+    try {
+        String[] versionSplit = originalTag.split("\\.")
+        return "${versionSplit[0]}.${versionSplit[1]}"
+    } catch (error) {
+        echo "[INFO] ${originalTag} cannot be reduced to the format X.Y"
+    }
+    return ''
```

Comment:
is there any reason behind returning `''` instead of throwing the error?

Reply:
I did not want to be blocking. But indeed, let's keep the decision on the "user" side. Gonna update that